candy-machine/program/tests/initialize.rs

```diff
@@ -54,3 +76,140 @@ async fn init_default_success() {
         .mint_and_assert_successful(context, Some(1), true)
         .await;
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn bot_tax_on_gatekeeper() {
+    let mut context = candy_machine_program_test().start_with_context().await;
+    let context = &mut context;
+
+    let gatekeeper_network = Keypair::new();
+    let gatekeeper_authority = Keypair::new();
+
+    let client = RpcClient::new("https://metaplex.devnet.rpcpool.com".to_string());
+
+    let gateway_account_pubkey =
+        Pubkey::from_str("gatem74V238djXdzWnJf94Wo1DcnuGkfijbf3AuBhfs").unwrap();
```

Comment:
Nit/cool thing to know:
You can replace all the 

`Pubkey::from_str("constant").unwrap()`

with the `pubkey` macro:

```rust
use solana_program::pubkey;
...
let gateway_account_pubkey = pubkey!("gatem74V238djXdzWnJf94Wo1DcnuGkfijbf3AuBhfs");
 ```
 
 For tests it doesn't really matter so you don't need to change this if you don't want to, but compile time checking this for prod code is much better. (We don't do this consistently in our codebase yet, but we should.)